src/cache_cf.cc

```diff
@@ -3912,8 +3912,9 @@ configFreeMemory(void)
     free_all();
     Dns::ResolveClientAddressesAsap = false;
     delete Config.ssl_client.defaultPeerContext;
+    delete Config.ssl_client.sslContext;
     Config.ssl_client.defaultPeerContext = nullptr;
```

Comment:
Please keep delete/assignment paired.

```suggestion
    Config.ssl_client.defaultPeerContext = nullptr;
    delete Config.ssl_client.sslContext;
```

Reply:
Done.